src/muya/lib/parser/marked/renderer.js

```diff
@@ -37,7 +37,8 @@ Renderer.prototype.emoji = function (text, emoji) {
   }
 }
 
-Renderer.prototype.code = function (code, lang, escaped, codeBlockStyle) {
+Renderer.prototype.code = function (code, infostring, escaped, codeBlockStyle) {
```

Comment:
Are we going to abandon the text after the space? In other words, after parsing, the text after the space will be lost?

Reply:
Yes, that's true but we are using the muya renderer in `importMarkdown.js` and we don't need the additional information. According to CommonMark/GFM Spec and https://github.github.com/gfm/#example-112:

> An info string can be provided after the opening code fence. Opening and closing spaces will be stripped, and the first word, prefixed with language-, is used as the value for the class attribute of the code element within the enclosing pre element.

And Github does the same. In the following example (`cpp python`) is the `c++` syntax highlighting used only:

```cpp python 
size_t bar = 0;

for i, name in enumerate(foo):
    print "iteration {iteration} is {name}".format(iteration=i, name=name)
```

Reply:
ok